content/en/docs/features/signed-images.md

```diff
@@ -8,4 +8,166 @@ tags:
 - images
 ---
 
-TODO
+
+
+# Overview
+[Encrypted images](/docs/features/encrypted-images/) provide confidentiality,
+but they do not provide _authenticity_ or _integrity_. Image signatures provide
+this additional property, preventing certain types of image tampering,
+[for example](https://docs.sigstore.dev/about/overview/#why-cryptographic-signing).
+
+In this brief guide, we show two tools that can be used to sign container images:
+[cosign](https://github.com/sigstore/cosign) and
+[skopeo](https://github.com/containers/skopeo). The skopeo tool can be
+used to to create both cosign signatures or "simple signatures" (which leverage
+gpg keys). For our purposes, our skopeo examples will use the simple signing
+approach. In any case, the general approach is to (1) create keys for signing,
+(2) sign a newly tagged image, and (3) update the
+[KBS](/docs/attestation/key-broker-service) with the public signature key and a
+security policy.
+
+# Creating an Image
+
+## Creating Keys
+
+### cosign
```

Comment:
Maybe it's worth to introduce tabs for Cosign & Skopeo GPG approaches - it will be separated and more visible for the user to use entire path from chosen type of signing

Reply:
:eyes: I'm worried those sections have too much content (rather than just a command or two for a tab). But making it clearer that the user should pick one of the two options is a good point. I'll add something above the subsections to this effect.

Reply:
@pawelpros Does [this](https://github.com/confidential-containers/confidentialcontainers.org/compare/f18657d8045bf4e9e91f1d930c2b80e6d0711972..98da73e6af76fb126395a7f58b189d2f2d309873) help?

Reply:
I was thinking rather about below tabs view:

`
{{< tabpane text=true right=true persist=header >}}

{{% tab header="Cosign" %}}
//Cosign content
{{% /tab %}}

{{% tab header="Skopeo" %}}
//Scopeo content
{{% /tab %}}

{{< /tabpane >}}
`